src/uwtools/drivers/forecast.py

```diff
@@ -162,11 +162,19 @@ def job_card(self):
         Turns the resources config object into a batch card for the configured Task.
         """
 
+    def run_cmd(self, run_cmd: str, *args, exec_name: str, **kwargs) -> str:
```

Comment:
It seems that there is a census that removing **kwargs would be the best option due to the unpredictably of the args needing 1 or 2 dashes. 

To my understanding, I think it would be best to remove **kwargs to not overcomplicate what arguments need to be passed in with keywords and just leave it at *args. This way the only keyword argument we have is exec_name.

Speaking of the only keyword argument being exec_name, I realize that the other required argument is run_cmd which currently if a user did not supply the run_cmd the tests would still pass. Just confirming that we have something more like:
`FV3Forecast().run_cmd("--export=ALL" run_cmd="mpiexec", exec_name="test_exec.py")`
Adding run_cmd towards the end due to the keywords positional requirement. Then the concatenation of the run command will look like `mpiexec --export=ALL test_exec.py`

If everyone is ok with removing **kwargs I will push my new changes and get some more feedback on these updates.